docker-images/kafka-mirror-maker/scripts/kafka_mirror_maker_tls_prepare_certificates.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Probably idea for different PR. But don't we have this script in every other Docker image?

Reply:
Agree. Anyway, they are slightly different. I think that we can at least move the `create_truststore` and `create_keystore` functions in a common script file and then include it.

Reply:
I agree we should be able to do something to modularise this once I've merged the certificate renewal work. I also wonder if there's any value in providing a base image for users writing Kafka applications, because setting up the key and truststores correctly is something which users need to do too. But maybe it's enough to cover that in the documentation.